CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+<h2>Data Format</h2>
+The file will be in `.json` format and the format of the data will be as follows _(Extra fields may be added later)_;
+
+```json
+[
+{
+ "name": "Awesome Company",
+ "establishedDate": "2022-11-01",
+ "location": "Fully remote",
+ "website": "https://anawesomewebsite.com",
+ "socials": {
+   "facebook": "https://facebook.com",
+   "instagram": "https://instagram.com",
+   "github": "https://github.com",
+   .
+   .
+   .
+   // Social media handles will go here
+   }
+},
+.
+.
+.
+// Several Companies here...
+]
+```
+
```

Comment:
Add a mention that keys not provided here can be added as well, but it's best if it's discussed with the team through the discussions tab or our discord server 🙏

Reply:
That should be discussed with the team so lets not do that right now!

Reply:
That sounds good.
But then maybe we should mention all the keys as discussed in #1 ??

Reply:
> That sounds good. But then maybe we should mention all the keys as discussed in #1 ??

I'll list those right away!